truffle-config.js

```diff
@@ -1,7 +1,7 @@
 const truffleConfig = require("@gnosis.pm/util-contracts/src/util/truffleConfig")
 
 const DEFAULT_GAS_PRICE_GWEI = 25
-const DEFAULT_GAS_LIMIT = 6721975
+const DEFAULT_GAS_LIMIT = 8e6
```

Comment:
I'm not sure I understand this change. Do you mind explaining?

Reply:
I believe this is the block gas limit assigned to the ganache instance. Our transaction was consuming more than an entire block, so we had to up the limit.

Reply:
OHHHHHH I read that in hex

Reply:
The real mainnet limit is 10e6